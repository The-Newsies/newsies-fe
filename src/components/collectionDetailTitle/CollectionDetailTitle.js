import React from 'react';
import PropTypes from 'prop-types';

export default function CollectionDetailTitle({ name, description }) {
  return (
    <div>
      <h3>{name}</h3>
      <p>{description}</p>
    </div>
  );
}

CollectionDetailTitle.propTypes = {
  name: PropTypes.string.isRequired,
  description: PropTypes.string
};
