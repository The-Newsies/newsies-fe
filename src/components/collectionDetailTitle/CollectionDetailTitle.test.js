import React from 'react';
import { shallow } from 'enzyme';
import CollectionDetailTitle from './CollectionDetailTitle';

describe('CollectionDetailTitle component', () => {
  it('renders CollectionDetailTitle', () => {
    const wrapper = shallow(<CollectionDetailTitle name='title' description='were' />);
    expect(wrapper).toMatchSnapshot();
  });
});
