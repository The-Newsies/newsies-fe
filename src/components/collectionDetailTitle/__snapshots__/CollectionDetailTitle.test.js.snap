// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionDetailTitle component renders CollectionDetailTitle 1`] = `
<div>
  <h3>
    title
  </h3>
  <p>
    were
  </p>
</div>
`;
